src/main/java/org/jboss/aerogear/keycloak/metrics/MetricsFilter.java

```diff
@@ -23,6 +23,8 @@
     private static final String METRICS_REQUEST_TIMESTAMP = "metrics.requestTimestamp";
     private static final MetricsFilter INSTANCE = new MetricsFilter();
 
+    private static final boolean URI_METRICS_ENABLED = Boolean.getBoolean("URI_METRICS_ENABLED");
```

Comment:
Not a JAva guy, so trying to understand would it default to false, in case the variable is not defined?

Reply:
according to the docs:

```
Returns true if and only if the system property named by the argument exists and is equal to the string "true". (Beginning with version 1.0.2 of the JavaTM platform, the test of this string is case insensitive.) A system property is accessible through getProperty, a method defined by the System class.

If there is no property with the specified name, or if the specified name is empty or null, then false is returned.
```

Reply:
Yeah, if the value is set and set to true it will return true, otherwise it returns false 
https://www.tutorialspoint.com/java/lang/boolean_getboolean.htm